src/solvers/dg_unstructured_quad/mappings_geometry_curved_2d.jl

```diff
@@ -97,40 +97,40 @@ end
 
 # construct the normals, and their normalization scalings for a curved sided element
 function calc_normals_and_scaling!(normals, scaling, element, nodes,
-                                   gamma_curves::AbstractVector{<:GammaCurve})
+                                   surface_curves::AbstractVector{<:CurvedSurface})
 
   # normals and boundary information for the left (local side 4) and right (local side 2)
   for j in eachindex(nodes)
     # side 2
-    X_xi, X_eta, Y_xi, Y_eta = transfinite_quad_map_metrics(1.0, nodes[j], gamma_curves)
+    X_xi, X_eta, Y_xi, Y_eta = transfinite_quad_map_metrics(1.0, nodes[j], surface_curves)
     Jtemp = X_xi * Y_eta - X_eta * Y_xi
-    scaling[j, 2, element]     = sqrt(Y_eta * Y_eta + X_eta * X_eta)
-    normals[1, j, 2, element]  = sign(Jtemp) * ( Y_eta / scaling[j, 2, element])
-    normals[2, j, 2, element]  = sign(Jtemp) * (-X_eta / scaling[j, 2, element])
+    scaling[j, 2, element]    = sqrt(Y_eta * Y_eta + X_eta * X_eta)
```

Comment:
```suggestion
    scaling[j, 2, element]    = sqrt(Y_eta^2 + X_eta^2)
```
That has the same performance (since `2` is known at compile time) but is more readable, at least for me.